backbone.localStorage.js

```diff
@@ -56,7 +56,7 @@ function result(object, property) {
 // with a meaningful name, like the name you'd give a table.
 // window.Store is deprectated, use Backbone.LocalStorage instead
 Backbone.LocalStorage = window.Store = function(name, serializer) {
-  if( !this.localStorage ) {
+  if( !this.localStorage() ) {
```

Comment:
I'm not sure that's right. Given it's `undefined`, in an unsupported browser, this will throw an error (which is not being caught here).